plutus-tx-plugin/executables/profile/Main.hs

```diff
@@ -24,12 +41,117 @@ import qualified PlutusCore.Default        as PLC
 
 import           Control.Lens.Combinators  (_2)
 import           Control.Lens.Getter       (view)
+import           Data.List                 (intercalate)
+import           Data.Maybe                (fromJust)
 import           Data.Proxy                (Proxy (Proxy))
 import           Data.Text                 (Text)
+import           Data.Time.Clock           (NominalDiffTime, UTCTime, diffUTCTime)
 import           Prettyprinter.Internal    (pretty)
 import           Prettyprinter.Render.Text (hPutDoc)
 import           System.IO                 (IOMode (WriteMode), withFile)
 
+data Stacks
+  = MkStacks
+  { -- | The variable name.
+    varName           :: String,
+    -- | The time when it starts to be evaluated.
+    startTime         :: UTCTime,
+    -- | The time spent on evaluating the functions it called.
+    timeSpentCalledFn :: NominalDiffTime
+  }
+  deriving (Show)
+
+-- | Write the time log of a program to a file in
+-- the plutus-tx-plugin/executables/profile/ directory.
+writeLogToFile ::
```

Comment:
I think this will end up being nicer with what I'm doing to `uplc`. Then we can have a workflow like:
1. Compile your program with `profile-all` and `dump-plc`
2. Run the dumped program with `uplc --trace-mode BLAH --log-output logs`
3. Run `log-to-stacks logs | flamegraph.pl`

Reply:
I changed the executable to `logToStacks` and it takes a list of file paths and process them to stacks. Updated the instructions also. Lmk what you think.